File: samples/go/src/models/lead.go
package models

type Lead struct {
	LeadOrigin      string `json:"leadOrigin"`
	Timestamp       string `json:"timestamp"`
	OriginLeadId    string `json:"originLead"`
	OriginListingId string `json:"originListingId"`
	ClientListingId string `json:"clientListingId"`
	Name            string `json:"name"`
	Email           string `json:"email"`
	DDD             string `json:"ddd"`
	Phone           string `json:"phone"`
	Message         string `json:"message"`
}

File: samples/java/src/main/java/br/com/olx/leadIntegration/models/Lead.java
package br.com.olx.leadIntegration.models;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Lead {
    private String leadOrigin;
    private String timestamp;
    private String originLeadId;
    private String originListingId;
    private String clientListingId;
    private String name;
    private String email;
    private String ddd;
    private String phone;
    private String message;
}